CHANGELOG.md

```diff
@@ -7,7 +7,8 @@
 - [tasks] add support for workspace-scoped task configurations. [#8917](https://github.com/eclipse-theia/theia/pull/8917)
 - [workspace] add support for configurations outside the `settings` object and add `WorkspaceSchemaUpdater` to allow configurations sections to be contributed by extensions. [#8917](https://github.com/eclipse-theia/theia/pull/8917)
 - [property-view] added initial version of a selection-based property-view [#8655](https://github.com/eclipse-theia/theia/pull/8655)
-  - A default implementation is available for file selections (via file navigator and default editors).
+- A default implementation is available for file selections (via file navigator and default editors).
+- [debug] added support for managing debug sessions for extensions from debug panel (previously only possible using `Hosted Plugin` commands) [#8706](https://github.com/eclipse-theia/theia/pull/8706)
```

Comment:
this probably should move to 1.13.0 now

Reply:
Obviously will rebase if we are approving this PR.

Reply:
Fixed